docs/conf.py

```diff
@@ -53,8 +54,10 @@
 modindex_common_prefix = ["template_project."]
 
 # html theme options
-html_static_path = ["_static"]
-html_logo = "_static/images/logo.png"
+html_title = f"{project} {release}"
+# html_logo = "_static/images/logo.png"
+html_theme = "qiskit-ecosystem"
+html_theme_options = {"sidebar_hide_name": False}
```

Comment:
This is the default. I think it's probably better to remove this line and have people refer to the Readme. 

I'm a big fan of not including default values in config because they are noisy. Presumably the default was set for a good reason. It's useful when you can go to a short config file and see exactly what is being changed from the default experience.